docs/design/design-reviews/recipes/engineering-feasibility-spikes.md

```diff
@@ -0,0 +1,92 @@
+# Engineering Feasibility Spikes: identifying and mitigating risk
+
+## Introduction
+
+Oftentimes engagements come with significant risks. Even after envisioning and Architectural Design Session (ADS), the engagement may still have substantial unknowns. These types of engagements warrant an exploratory/validation phase where engineering feasibility spikes can be conducted to de-risk the engagement, bring clarity, and help drive a sound architecture.
+
+**Engineering feasibility spikes** are a regimented yet collaborative feedback loop to capitalize on individual learnings to inform the team and increase the team’s knowledge and understanding while minimizing engagement risks. These spikes can be conducted during an exploratory/investigate phase immediately after the ADS and before engineering sprints.
+
+The following guidelines outline how Microsoft and the customer can incorporate engineering feasibility spikes into the day-to-day agile processes. It describes how to prioritize the spikes, set guardrails around this process and capitalize on associated benefits.
+
+## Background
```

Comment:
Suggest removing this section or integrating into the Introduction. In the other sections and recipes there is often a "Goals" section that lays out the high level of what the process aims to accomplish, perhaps consider introducing one or converting Intro more to Goals?